kld/src/ldk/mod.rs

```diff
@@ -57,6 +59,7 @@ pub(crate) type LiquidityManager = ldk_lsp_client::LiquidityManager<
     Arc<KeysManager>,
     Arc<KldLogger>,
     Arc<KeysManager>,
+    Arc<DummyFilter>,
```

Comment:
you can use `Arc<dyn Filter + Send + Sync>` as we just set the filter to None anyway. Then there is no need for the DummyFilter.

Reply:
Good idea.